Content.Shared/_RMC14/Xenonids/Pierce/XenoPierceSystem.cs

```diff
@@ -41,7 +44,7 @@ public override void Initialize()
 
     private void OnXenoPierceAction(Entity<XenoPierceComponent> xeno, ref XenoPierceActionEvent args)
     {
-        if (args.Handled)
+        if (args.Handled || !_timing.IsFirstTimePredicted)
```

Comment:
Is this necessary? I have it fixed on my branch as well and it didn't seem like it